docs/tutorial/extensions.md

```diff
@@ -513,3 +513,33 @@ assert circuit.encrypt_run_decrypt(0, 3, -5) == -5
 {% hint style="info" %}
 `fhe.if_then_else` is just an alias for [np.where](https://numpy.org/doc/stable/reference/generated/numpy.where.html).
 {% endhint %}
+
+## fhe.identity(value)
+
+Allows you to copy the value:
+
+```python
+import numpy as np
+from concrete import fhe
+
+@fhe.compiler({"x": "encrypted"})
+def f(x):
+    return fhe.identity(x)
+
+inputset = [np.random.randint(-10, 10) for _ in range(10)]
+circuit = f.compile(inputset)
+
+assert circuit.encrypt_run_decrypt(0) == 0
+assert circuit.encrypt_run_decrypt(1) == 1
+assert circuit.encrypt_run_decrypt(-1) == -1
+assert circuit.encrypt_run_decrypt(-3) == -3
+assert circuit.encrypt_run_decrypt(5) == 5
+```
+
+{% hint style="info" %}
+Identity extension can change the bit-width of the input.
```

Comment:
That's not clear to me, can you elaborate?

Reply:
It allows you to do 3b -> 5b for example. Keeping the same value.

Reply:
ok so foor me this sentence have the meaning that it can have an effect on the bit width assignment on the input subgraph. Maybe to clarify we can say the input and output bitwitdh of the identity may differ and give an example?

Reply:
Done.